src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -114,7 +114,7 @@ export const DEFAULT_LOGGER: Logger = {
   error: (...args) => console.error(args),
 };
 
-export function getDefaultSimulatedRelayerAddress(chainId?: number) {
```

Comment:
Renamed because this function had a very long name and often caused line wrapping (especially because it's often used to set default vales in function interfaces).